packages/@lwc/perf-benchmarks/package.json

```diff
@@ -23,10 +23,9 @@
         "@lwc/synthetic-shadow": "5.0.0"
     },
     "devDependencies": {
-        "@best/cli": "^9.0.1",
-        "@best/runner-headless": "^9.0.1",
-        "@best/runner-remote": "^9.0.1",
-        "@lwc/rollup-plugin-node-resolve-legacy": "npm:rollup-plugin-node-resolve@5.2.0",
+        "@best/cli": "^10.0.0-alpha.0",
+        "@best/runner-headless": "^10.0.0-alpha.0",
+        "@best/runner-remote": "^10.0.0-alpha.0",
```

Comment:
@nolanlawson Note: I'll update this PR once v10.0.0 is released (currently checking to see how different projects handle this change).